app/views/members/index.html.slim

```diff
@@ -3,21 +3,20 @@
     = link_to '<< BACK', @group
     
 .row
-  .col-lg-6
+  .col-xs-12.col-md-6.col-lg-6
```

Comment:
Yes I know... My mistake for doing it desktop-first and forgetting to remove le lg class :) I will fix this before merging.